parsl/channels/base.py

```diff
@@ -81,12 +81,6 @@ def pull_file(self, remote_source: str, local_dir: str) -> str:
         '''
         pass
 
-    @abstractmethod
-    def close(self) -> None:
-        ''' Closes the channel.
-        '''
-        pass
-
```

Comment:
This hunk ... is debatable.  Channels are going away anyway, so "whatever."

Reply:
removing it helps flush out where it was called for the typechecker. sometimes.

my thinking mostly is that I would like to remove all the bits of the base channel class piecewise, eventually leaving an empty class that can be removed.